extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -187,10 +194,7 @@ public function searchVideosByTitle() {
 		if ( empty( $title ) ) {
 			throw new Exception( "Please include a value for 'title'." );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS112)